service/gcs/bridge/bridge.go

```diff
@@ -652,16 +651,16 @@ func (b *Bridge) execProcess(w ResponseWriter, r *Request) {
 }
 
 func (b *Bridge) killContainer(w ResponseWriter, r *Request) {
-	b.signalContainer(w, r, oslayer.SIGKILL)
+	b.signalContainer(w, r, syscall.SIGKILL)
```

Comment:
It seems that `unix` is the preferred package instead of `syscall` these days.

Reply:
Agreed. And this isnt multiplat so I will switch.